### DISCUSSION TITLE
「开源之史」系列之四：软件的知识财产法律与许可的演化 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#15] [Announcements]
# 「开源之史」系列之四：软件的知识财产法律与许可的演化 - 开源之道

https://opensourceway.community/posts/history-of-open-source/02-ip-law-and-license-evolution/

**Comment by ZoomQuiet** [comment]
#是也乎
最后大头像太影响留言心情了, 推荐去除....

其实吧, 纵观人类财产史, 可以有更加简洁的归纳:
随着技术的进步, 凡是可以转化为资本的事物, 都在逐一被产权吞噬...

现在可以看出来的倾逈:

- NDA 已经成为产权关注物
- 思想本身也将成为
- 意识在进入 元宇宙 后, 也必然分解为必要产权物
- 灵魂呢? 嘦可以识别并标定出来, 也必须可以
- ...

所以, 互惠性, 可能也是 GPL 创造的最可贵思想,
为以后可以深入灵魂的占领, 给出一条法律兼容的抗争方向.

**Reply by lijiangsheng1** [reply]
继续深入挖掘斗争～ 看许可的条款，也能看出人类自我毁灭的倾向。

**Reply by ZoomQuiet** [reply]
#是也乎
应该说 资本 的贪婪在逐步侵食 资产 的概念,
从以往一定要有物质附着物才能算资产,
到后来思想/间接思想记载/... 都要加以拥有, 才能进一步开发出资本衍生物...
